chapters/object_types/spatial_fields.txt

```diff
@@ -53,3 +53,53 @@ Structured regular fields only need to store the values of the samples, because
 their addresses in memory can be easily computed from a 3D position. A
 common type of structured spatial fields are regular grids.
 ====
+
+[[object_types_spatial_field_unstructured]]
+==== Unstructured 
+
+Extension `KHR_SPATIAL_FIELD_UNSTRUCTURED`
+
+Unstructured spatial fields are created by passing the subtype string
+`unstructured` to <<object_types_spatial_field, `anariNewSpatialField`>>. The parameters
+understood by unstructured spatial fields are summarized in the table below.
+
+.Configuration parameters for unstructured spatial fields.
+[cols="<,<2,>,<5",options="header,unbreakable"]
+|===================================================================================================
+| Name            | Type                       | Description
+| vertex.data     |`ARRAY1D` of `UFIXED8` / `FIXED16` / `UFIXED16` / `FLOAT32` / `FLOAT64` | array 
+                                          of vertex values to be sampled. The size and value type of
+                                          the spatial field is inferred from `vertex.data`
+| vertex.position |`ARRAY1D` of `FLOAT32_VEC3` | array of vertex positions 
+| index           |`UINT32` / `UINT64`         | array of indices into the vertex arrays that form cells 
+| cell.data       |`ARRAY1D` of `UFIXED8` / `FIXED16` / `UFIXED16` / `FLOAT32` / `FLOAT64` | array of
+                                          cell values to be sampled. The size and value type of the
+                                          spatial field is inferred from `cell.data`
+| cell.type       |`UINT8`                     | array of cell types where 0 == TETRAHEDRON, 1 == HEXAHEDRON, 2 == WEDGE
```

Comment:
Yeah...I think an enum type is appropriate here. And as you pointed out it should be `ARRAY1D` of `UINT8`. I'm not sure if matching VTK's definition is important assuming there are other APIs that you may want to use that have their own set of values. Using the enum instead of the raw value should be more portable across different APIs.

Reply:
Are there other libs than VTK that work with Unstructured?